src/ui/Intro.tsx

```diff
@@ -41,22 +40,55 @@ export function Intro({ children }: IntroProps): JSX.Element {
       <Suspense fallback={<Ready setReady={setReady} />}>{children}</Suspense>
       <div className={`fullscreen bg ${ready ? 'ready' : 'notready'} ${clicked && 'clicked'}`}>
         <div className="stack">
-          <div className="intro-keys">
-            <Keys style={{ paddingBottom: 20 }} />
-            <a className="continue-link" href="#" onClick={() => ready && setClicked(true)}>
-              {!ready ? <Loader /> : 'Click to continue'}
-            </a>
-          </div>
           {session?.user?.aud !== 'authenticated' ? (
             <Auth />
           ) : (
-            <div>
-              Hello {session.user.user_metadata.full_name}
-              <button className="logout" onClick={unAuthenticateUser}>
-                Logout
-              </button>{' '}
+            <div className="intro-car-select">
```

Comment:
The vercel preview isn't showing the car colour select.

Regardless, I would add this feature to the 'editor' menu (keyboard option: .)
It would be even better if we had a colour picker in the vehicle tab of the editor menu! That way the users' aren't locked to only 3 colours.

Reply:
I think this belongs on the title screen, the editor seems to be for advanced users who want to do experimental things with physics.